bitnami/etcd/values.yaml

```diff
@@ -462,24 +467,24 @@ service:
   ## clusterIP: None
   ##
   clusterIP: ""
-  ## @param service.port etcd client port
+  ## @param service.ports.client etcd client port
+  ## @param service.ports.peer etcd peer port
+  ##
+  ports:
+    client: 2379
+    peer: 2380
```

Comment:
These changes are affecting other parts that you didn't take into account (apart from `svc.yaml`):
```console
$ ag "service.port" bitnami/etcd/templates
bitnami/etcd/templates/NOTES.txt
42:etcd can be accessed via port {{ .Values.service.port }} on the following DNS name from within your cluster:
48:    kubectl run {{ template "common.names.fullname" . }}-client --restart='Never' --image {{ template "etcd.image" . }}{{- if .Values.auth.rbac.enabled }} --env ROOT_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "common.names.fullname" . }} -o jsonpath="{.data.etcd-root-password}" | base64 --decode){{- end }} --env ETCDCTL_ENDPOINTS="{{ template "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}:{{ .Values.service.port }}" --namespace {{ .Release.Namespace }} --command -- sleep infinity
71:    echo "etcd URL: http://$SERVICE_IP:{{ .Values.service.port }}/"
75:    kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "common.names.fullname" . }} {{ .Values.service.port }}:{{ .Values.service.port }} &
76:    echo "etcd URL: http://127.0.0.1:{{ .Values.service.port }}"
99:    export ETCDCTL_ENDPOINTS=https://{{ template "common.names.fullname" . }}-0:{{ .Values.service.port }}

bitnami/etcd/templates/statefulset.yaml
161:              value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME).{{ $etcdHeadlessServiceName }}.{{ .Release.Namespace }}.svc.{{ $clusterDomain }}:{{ .Values.containerPorts.client }},{{ $etcdClientProtocol }}://{{ $etcdFullname }}.{{ .Release.Namespace }}.svc.{{ $clusterDomain }}:{{ .Values.service.port }}"
$ ag "service.peerPort" bitnami/etcd/templates/
bitnami/etcd/templates/svc.yaml
41:    - name: {{ default "peer" .Values.service.peerPortNameOverride | quote }}
42:      port: {{ .Values.service.peerPort  }}

bitnami/etcd/templates/svc-headless.yaml
31:    {{- if .Values.service.peerPortNameOverride }}
33:    - name: {{ .Values.service.peerPortNameOverride }}-ssl
35:    - name: {{ .Values.service.peerPortNameOverride }}
```

Reply:
Thanks you Juan, I forgot to fix the references to `service.port`